libkbfs/md_ops.go

```diff
@@ -563,6 +563,13 @@ func (md *MDOpsStandard) PruneBranch(
 	return md.config.MDServer().PruneBranch(ctx, id, bid)
 }
 
+// ResolveBranch implements the MDOps interface for MDOpsStandard.
+func (md *MDOpsStandard) ResolveBranch(
```

Comment:
seems clunky to me to have an `MDOps` interface method that some implementations don't implement. Perhaps define a separate interface, like, `BranchResolver`, and then callers that have an `MDOps` object can check via type assertion?